src/wireviz/wv_graphviz.py

```diff
@@ -0,0 +1,632 @@
+                if conn.from_ is not None:
+                    ins.append(str(conn.from_))
+                if conn.to is not None:
+                    outs.append(str(conn.to))
+
+        cells_above = [
+            Td(" " + ", ".join(ins), align="left"),
+            Td(" "),  # increase cell spacing here
+            Td(bom_bubble(wire.bom_id)) if cable.category == "bundle" else None,
+            Td(":".join([wi for wi in wireinfo if wi is not None and wi != ""])),
+            Td(" "),  # increase cell spacing here
+            Td(", ".join(outs) + " ", align="right"),
+        ]
+        cells_above = [cell for cell in cells_above if cell is not None]
+        rows.append(Tr(cells_above))
+
+        # the wire itself
+        rows.append(Tr(gv_wire_cell(wire, len(cells_above))))
+
+        # row below the wire
+        if wire.partnumbers:
+            cells_below = partnumbers2list(
+                wire.partnumbers, parent_partnumbers=cable.partnumbers
+            )
+            if cells_below is not None and len(cells_below) > 0:
+                table_below = (
+                    Table(
+                        Tr([Td(cell) for cell in cells_below]),
+                        border=0,
+                        cellborder=0,
+                        cellspacing=0,
+                    ),
+                )
+                rows.append(Tr(Td(table_below, colspan=len(cells_above))))
+
+    rows.append(Tr(Td("&nbsp;")))  # spacer row on bottom
+    tbl = Table(rows, border=0, cellborder=0, cellspacing=0)
+    return tbl
+
+
+def gv_wire_cell(wire: Union[WireClass, ShieldClass], colspan: int) -> Td:
+    if wire.color:
+        color_list = ["#000000"] + wire.color.html_padded_list + ["#000000"]
+    else:
+        color_list = ["#000000"]
+
+    wire_inner_rows = []
+    for j, bgcolor in enumerate(color_list[::-1]):
+        wire_inner_cell_attribs = {
+            "bgcolor": bgcolor if bgcolor != "" else "#000000",
+            "border": 0,
+            "cellpadding": 0,
+            "colspan": colspan,
+            "height": 2,
+        }
+        wire_inner_rows.append(Tr(Td("", **wire_inner_cell_attribs)))
+    wire_inner_table = Table(wire_inner_rows, border=0, cellborder=0, cellspacing=0)
+    wire_outer_cell_attribs = {
+        "border": 0,
+        "cellspacing": 0,
+        "cellpadding": 0,
+        "colspan": colspan,
+        "height": 2 * len(color_list),
+        "port": f"w{wire.index+1}",
+    }
+    # ports in GraphViz are 1-indexed for more natural maping to pin/wire numbers
+    wire_outer_cell = Td(wire_inner_table, **wire_outer_cell_attribs)
+
+    return wire_outer_cell
+
+
+def gv_edge_wire(harness, cable, connection) -> Tuple[str, str, str, str, str]:
+    if connection.via.color:
+        # check if it's an actual wire and not a shield
+        color = f"#000000:{connection.via.color.html_padded}:#000000"
+    else:  # it's a shield connection
+        color = "#000000"
```

Comment:
Back in the day, drawing wires as thin when they don't have an explicit color assigned, was intentional, to distinguish an "uncolored" wire from a white or black one. For example, shields are also drawin in this style, since they don't have a color per se.

Reply:
OK, didn't know that.